src/platform/plugins/shared/workflows_management/common/schema.ts

```diff
@@ -69,6 +69,15 @@ const connectors = [
       },
     ],
   },
+  {
+    type: 'wait',
+    params: [
+      {
+        name: 'duration',
+        type: 'string' as const,
+      },
+    ],
+  },
```

Comment:
```suggestion
```
No need to declare connector here. Also, `delay` type should be removed in favor of your new step. You already declared the wait step in `src/platform/packages/shared/kbn-workflows/spec/schema.ts`,  include it in `createRecursiveStepSchema` https://github.com/Kiryous/kibana/blob/main/src/platform/packages/shared/kbn-workflows/spec/lib/generate_yaml_schema.ts#L56-L80